### DISCUSSION TITLE
Rule to compare 2 transactions by amount? Credit card payments = 2 transactions.

### DISCUSSION THREAD

**Discussion by Plutoaurus** [firefly-iii/firefly-iii#7861] [Firefly III]
My bank reports credit card payments as double transactions.
1 withdraw from current accout
1 import to card account

Annoylingly, my bank doesn't name both transactions the same.
1 transaction is just DD Payment, so not really helpful.
The reference nordigen imports doesn't match the DD repayment.

Is there a way to automate
If 2 transactions with same date and amount, then do delete one.

**Comment by M1scer** [comment]
I have a simmilar issue with transfers. I do automatic import via nordigen of two accounts.
If i transfer money between them they will listed twice -> each import creates an transfer.

It would be nice to compare two transactions to delete the duplicates automaticaly.

**Comment by JC5** [comment]
This is not something Firefly III can solve easily, because "duplicates" are never really duplicates. If they were exactly the same, Firefly III wouldn't import them. 

For this, you'll have to tinker around with webhooks.
